### PR TITLE
Update descriptive lines

### PR DESCRIPTION
**Pull request by webolot**

### CODE REVIEW

File: po/ca.po [hunk 1]
```diff
@@ -0,0 +1,1483 @@
+# Catalan translations for solaar package.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
```
Comment (mschilli87) [inline po/ca.po:2]:
```suggestion
# Copyright (C) 2021 Marc Serra
```

Not sure if you'd rather hand over the copyright to Daniel Pavel (@pwr).

File: po/ca.po [hunk 2]
```diff
@@ -0,0 +1,1483 @@
+# Catalan translations for solaar package.
```
Comment (mschilli87) [inline po/ca.po:1]:
```suggestion
# Catalan translations for Solaar package.
```

File: po/ca.po [hunk 3]
```diff
@@ -0,0 +1,1483 @@
...
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
```
Comment (mschilli87) [inline po/ca.po:3]:
```suggestion
# This file is distributed under the same license as the Solaar package.
```

**Review by mschilli87 — COMMENTED** [review]
Just a few more suggestions.

**Comment by pfps** [comment]
As the original update has already been merged, you need to update your clone and then work from there.

**Comment by pfps** [comment]
Also, Solaar commit messages generally start with something saying what part of Solaar is being updated, so translation commit messages should look like "po: ...".

**Comment by pfps** [comment]
There are a couple of other translations that need similar updating.  I've updated the other two.

@webolot Can you update your two as a separate PR and give the commits names like "po: update XXX translation"?  I would like to send out version 1.1.0 in the near future with clean translations.

**Comment by pfps** [comment]
By the way, if you don't want your email on the translations just remove that line.

**Comment by webolot** [comment]
Thank's for all your comments and recomendations.

Give me few hours more (I need to finish an important project) and I will review all that you commented.

**Comment by webolot** [comment]
I hope I have done well now. If not, excuse my ignorance.

**Comment by pfps** [comment]
Working with git can have quite a learning curve.  This PR has the right outcome, but too many commits.  And because I prematurely merged you initial Catalan translation, this part of the PR has to be removed.

What I did was grab this PR and fix things up so that there are only two commits, each with a suitable message.  This preserves your authorship.

Anyway, take a look at PR #1342.  If that looks good to you, I'll merge it in.

**Comment by webolot** [comment]
@pfps of course it's for me. 

thank's!

**Comment by pfps** [comment]
OK, done.  And thanks for the new translations.

**Pull request closed without merging**